mcrouter/lib/carbon/example/gen/HelloGoodbyeService.thrift

```diff
@@ -16,14 +16,12 @@ include "thrift/annotation/cpp.thrift"
 include "mcrouter/lib/network/gen/Common.thrift"
 include "mcrouter/lib/carbon/example/gen/HelloGoodbye.thrift"
 
-include "common/fb303/if/fb303.thrift"
-
 cpp_include "mcrouter/lib/carbon/example/gen/HelloGoodbyeMessages.h"
 
 namespace cpp2 hellogoodbye.thrift
 namespace py3 hellogoodbye.thrift
 
-service HelloGoodbye extends fb303.FacebookService {
+service HelloGoodbye {
```

Comment:
Ugly, but installing `fb303` via `fbcode_builder` still doesn't help the Thrift compiler find `fb303` IDLs. Since this is a test/example service, perhaps this is okay.